include/faker-cxx/types/Sex.h

```diff
@@ -8,13 +8,52 @@ enum class Sex
     Female,
 };
 
-inline std::string toString(Sex sex)
+const std::map<Language, std::map<Sex, std::string>> sexTranslations = {
```

Comment:
Apply changes to Person::sex() to take language as an optional argument

Reply:
done